crates/path/src/path.rs

```diff
@@ -105,11 +105,11 @@ impl Path {
 
     /// Creates an Empty `Path`.
     #[inline]
-    pub fn new() -> Path {
+    pub fn new(num_attributes: usize) -> Path {
```

Comment:
Hi, thanks for the PR.

It's a bit early for API breaking changes and since most users of the crate don't use attributes a lot, I'd like the default to remain simple. Could you instead add another constructor `Path::with_attributes(usize) -> Path`?

Reply:
Done!